packages/dropdowns/src/elements/Multiselect/Multiselect.tsx

```diff
@@ -432,10 +444,3 @@ Multiselect.defaultProps = {
 };
 
 Multiselect.displayName = 'Multiselect';
-
-/**
- * @extends HTMLAttributes<HTMLDivElement>
- */
-export default withTheme(Multiselect) as React.FunctionComponent<
```

Comment:
All `withTheme` exports were failing to properly render Storybook prop sheets. This PR swaps out the preferred `styled-components` [ThemeContext](https://styled-components.com/docs/advanced#via-usecontext-react-hook).